swig/package.xml

```diff
@@ -6,6 +6,10 @@
   <maintainer email="levi.armstrong@swri.org">Levi Armstrong</maintainer>
   <license>Unknown</license>
 
+  <depend>automake</depend>
+  <depend>pcre</depend>
+  <depend>bison</depend>
```

Comment:
Are these needed at runtime as well?

If not, perhaps specify them as `build_depend`s.